### PR TITLE
Change font to vazir

### PR DESCRIPTION
**Pull request by Mazafard**
- add vazir font support just for persian characters.
- change height to past style for texts.
- fix auto direction in post, comment and search text area

### CODE REVIEW

**Review by Mazafard — COMMENTED** [review]
checked all files.

**Comment by indeyets** [comment]
Thank you!
I took auto-direction patch to "stable" manually (it will be available for testing on candy.freefeed.net soon).
We need to think about font addition a bit more. I'm not sure it is a good idea to force loading for all users. Maybe we could hide it behind a setting (not the most beautiful solution) or use some kind of language-detection feature here

**Comment by kukutz** [comment]
Maybe we should look at Accept-Language header?

File: styles/common/vazir.scss [hunk 1]
```diff
@@ -0,0 +1,72 @@
+@font-face {
+  unicode-range: U+0600-06ff;
```
Comment (Mazafard) [inline styles/common/vazir.scss:2]: @indeyets @indeyets It just load for persian characters, and if the character is not in this range: U+0600-06ff, automatically it use the next font.

**Comment by indeyets** [comment]
@Mazafard merged. thank you!
I did it a bit differently (imported font from NPM), but kept essence of your changes and commit is marked as authored by you (see link above this comment)

**Pull request closed without merging**